prow/external-plugins/cherrypicker/server.go

```diff
@@ -390,8 +391,26 @@ func (s *Server) handle(l *logrus.Entry, requestor string, comment *github.Issue
 		return err
 	}
 
-	// Checkout a new branch for the cherry-pick.
+	// New branch for the cherry-pick.
 	newBranch := fmt.Sprintf(cherryPickBranchFmt, num, targetBranch)
+
+	// Check if that branch already exists, which means there is already a PR for that cherry-pick.
+	if r.BranchExists(newBranch) {
+		// Find the PR and link to it.
+		prs, err := s.ghc.GetPullRequests(org, repo)
+		if err != nil {
+			return err
+		}
+		for _, pr := range prs {
+			if pr.Head.Ref == fmt.Sprintf("%s:%s", s.botName, newBranch) {
+				resp := fmt.Sprintf("Uh, looks like #%d has already been cherry picked in %s", num, pr.HTMLURL)
```

Comment:
Is the "Uh," prefix too casual?

Reply:
maybe... (I am not that good in English) what do you suggest?

Reply:
Unless you feel strongly about it, I think you can just drop it. "Looks like #%d..."

Reply:
You're right, I have just dropped it. Thanks @fejta !